### ISSUE TITLE
Invalid Loss error when training on the GPU

### ISSUE THREAD

**Issue by dongishan** [gm-spacagna/deep-ttf#6]
I've used your implementation of the WTTE-RNN on the CPU and managed to complete training without any issues. However, when I used the same implementation on a Nvidia GPU I am getting the Invalid Loss error at around 25/100 epochs.

Following are the only changes I've done made when running on the GPU:
1. CuDNNGRU is used instead of the GRU.

2. The masking layer is removed from the the model as CuDNNGRU does not support masking. I.e. following line is removed:
`model.add(Masking(mask_value=mask_value, input_shape=(None, n_features)))`

3. An initial input layer is added to the model:
`mode.add(InputLayer(input_shape(None, n_features)))`

The epsilon is 1e-10 and the Keras backend is Tensorflow. And the WTTE-RNN version is 1.1.1.

**Comment by archenroot** [comment]
@dongishan - have you managed to fix this or found root cause..?
